src/cryptography/x509/base.py

```diff
@@ -552,7 +554,9 @@ def __init__(
         self,
         subject_name: typing.Optional[Name] = None,
         extensions: typing.List[Extension[ExtensionType]] = [],
-        attributes: typing.List[typing.Tuple[ObjectIdentifier, bytes]] = [],
+        attributes: typing.List[
+            typing.Tuple[ObjectIdentifier, bytes, typing.Optional[int]]
```

Comment:
Why an integer and not an element from _ASN1Type like we did in NameAttribute?

Reply:
Oh, sure.

Reply:
actually on closer review on `Attribute` it's just an `int` from that enum

Reply:
Would you still like me to switch to ASN1Type?

Reply:
Yeah we extract the int from it but I’d still rather use the enum for the API